pygments/token.py

```diff
@@ -189,6 +189,7 @@ def string_to_tokentype(s):
     Operator.Word:                 'ow',
 
     Punctuation:                   'p',
+    Punctuation.Marker:            'p-Marker',
```

Comment:
I set the class name to "p-Marker" because apparently that is the class name already being used by Pygments. For example, if you run Pygments on [this test file](https://github.com/pygments/pygments/blob/766c0f562633273650798301361b43695db97911/tests/examplefiles/pytb/py3tb_test.py3tb), that's the class name it gives the `Punctuation.Marker` token in the returned HTML:

```shell
$ cat << EOF | pygmentize -l pycon -f html /dev/stdin
  File "<stdin>", line 1
    1+
     ^
SyntaxError: invalid syntax
EOF
<div class="highlight"><pre><span></span>  File <span class="nb">&quot;&lt;stdin&gt;&quot;</span>, line <span class="m">1</span>
    <span class="mi">1</span><span class="o">+</span>
     <span class="p p-Marker">^</span>
<span class="gr">SyntaxError</span>: <span class="n">invalid syntax</span>
</pre></div>
```
I couldn't seem to find anywhere else in the code that "p-Marker" is actually referenced, so I don't know how it's doing this. But that seems to be the class name it wants to use.

Reply:
Ah, I think I get it now. I can definitely shorten the name; "pm" makes sense to me. :+1: